push.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "pushingcc",
+    "email": "pushing@mailfence.com"
+  },
+
+  "record": {
+    "CNAME": "pushing.pages.dev"
+    "TXT": "dh=63d8f2f88286c81885273cdf5f07364b15884ff6"
```

Comment:
Cannot have CNAME and TXT records together.

Reply:
for doing discord txt verification check here https://www.is-a.dev/docs/discord_verification/

Reply:
Fixed and removed for now.